### PR TITLE
Add createCustomerCard and chargeCustomerCard routes

### PR DESCRIPTION
**Pull request by dzello**
These new routes are needed for the addition of card on file functionality to the Super Cookie example React Native application.

To avoid duplication, I pulled error message processing and sending into a helper.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,63 +72,124 @@ app.post('/chargeForCookie', async (request, response) => {
       `[Error] Status:${e.status}, Messages: ${JSON.stringify((JSON.parse(e.response.text)).errors, null, 2)}`);
 
     const { errors } = (JSON.parse(e.response.text));
+    sendErrorMessage(errors, response);
+  }
+});
 
-    switch(errors[0].code) {
-        case "UNAUTHORIZED":
```
Comment (okenshields) [inline index.js]: Why removing some the cases, e.g. "UNAUTHORIZED"?

File: index.js [hunk 2]
```diff
@@ -69,63 +72,124 @@ app.post('/chargeForCookie', async (request, response) => {
...
-          break;
-        default:
-          response.status(400).send({
-              errorMessage: "Payment error. Please contact support if issue persists."
-          });
-          break;
-    }
+app.post('/chargeCustomerCard', async (request, response) => {
+  const requestBody = request.body;
+  const locations = await locationsApi.listLocations();
```
Comment (okenshields) [inline index.js]: Suggest putting this in a try block as well

Comment (mootrichard) [reply index.js]: I accidentally merged before it loaded these comments, but do you mean to move this down in the below try block, or into its own?

(I will be creating another PR for these changes)

Comment (mootrichard) [reply index.js]: Nvm, I see how it is in the other endpoint. I can refactor it to match

**Pull request merged**